Sources/NIO/CircularBuffer.swift

```diff
@@ -23,7 +23,8 @@ public protocol AppendableCollection: Collection {
 /// An automatically expanding ring buffer implementation backed by a `ContiguousArray`. Even though this implementation
 /// will automatically expand if more elements than `initialRingCapacity` are stored, it's advantageous to prevent
 /// expansions from happening frequently. Expansions will always force an allocation and a copy to happen.
-public struct CircularBuffer<E>: CustomStringConvertible, AppendableCollection {
+public struct CircularBuffer<E>: CustomStringConvertible, AppendableCollection
```

Comment:
mind taking this change out, we generally have the curly braces on the same line